scripts/heat/instances.yaml

```diff
@@ -39,6 +39,16 @@ resources:
       flavor: { get_param: flavor }
       networks:
         - port: { get_resource: server_port }
+      user_data_format: RAW
+      user_data:
+        #!/bin/sh
+        cat >> /usr/bin/ping_forever << EOF
+        #!/bin/sh
+        trap "" 1 2 3 15
+        ping 8.8.8.8 | tee /var/log/ping_outside.out
```

Comment:
pinging the outside is worth checking. Note though that most complains are about [can't ping my VM](http://schd.ws/hosted_files/openstacksummitoctober2015tokyo/54/can't%20ping%20my%20vm_no_image.pdf) so we need to check that too. Probably from the crowbar node, since this is happening in our environments.

About the script format, I still don't have an opinion yet, but it's good that you found a way to add a script to the VM.